.eslintrc.json

```diff
@@ -1,4 +1,5 @@
 {
+    "root": true,
```

Comment:
Is this root as in root of the repo or root as in run with root permissions?

Reply:
Has nothing to do with the root user 😀

Eslint by default is looking into all parent directories for and eslintrc, and this just tells eslint to look no further. So it the dev has a `~/.eslintrc` it won't be considered.

https://eslint.org/docs/latest/use/configure/configuration-files#cascading-and-hierarchy

Reply:
Perfect :-) Thanks for the info!